src/dialogue/mod.rs

```diff
@@ -66,11 +66,11 @@ where
 impl<B, H, HI, HO, HE> Handler<HandlerInput> for DialogueDecorator<B, H, HI>
 where
     H: Handler<HI, Output = Result<HO, HE>> + Clone + 'static,
-    HI: TryFromInput,
+    HI: TryFromInput + Clone,
```

Comment:
Lower requirements by manual Clone implementation for DialogueDecorator

Reply:
What do you mean?

Reply:
https://github.com/rust-lang/rust/issues/26925